article/_posts/2019-07-18-Striving-for-Simplicity.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: review
+title: "Striving for Simplicity in Off-policy Deep Reinforcement Learning"
+tags: deep-learning reinforcement
+author: "Antoine Théberge"
+cite:
+    authors: "Rishabh Agarwal, Dale Schuurmans, Mohammad Norouzi"
+    title:   "Striving for Simplicity in Off-policy Deep Reinforcement Learning"
+    venue:   "Arxiv 2019"
+pdf: "https://arxiv.org/pdf/1907.04543"
+---
+
+
+# Highlights
+
+- Learning offline with DQN doesn't work really well
+- QR-DQN and C51 work better in the offline setting than DQN but are complicated
+- Ensemble-DQN and REM both outperform DQN and provide similar performance to distributional algorithms in the offline setting
+
+# Introduction
+
+Ever since the [Nature Deep Q-Network paper](https://www.nature.com/articles/nature14236), DQNs have been very promising for Deep Reinforcement Learning (DRL). However, their sample efficiency is often criticized; for example, requiring a couple of millions of frames to achieve human-level performance on Atari games. Offline (and off-policy) learning is a way to alleviate this problem by learning on a fixed set of samples collected from any policy. 
+
+Q-learning tries to maximize the expected cumulative discounted reward
+
+![](/article/images/striving-simplicity/eq1.jpeg)
+
+which can be defined recursively as
+
+![](/article/images/striving-simplicity/eq2.jpeg)
+
+with the optimal Q-function defined as
+
+![](/article/images/striving-simplicity/eq3.jpeg)
+
+Distributional RL makes use of a distribution over returns $$Z^{*}(s,a)$$ instead of the scalar function $$Q^{*}(s,a)$$. In this context, the Bellman optimality operator is defined as
+
+![](/article/images/striving-simplicity/eq5.jpeg)
+
+with either a categorical distribution supported on a fixed set of locations $$z_{1..k}$$ (C51) or a uniform mixture of $$K$$ Diracs (QR-DQN). Both are not necessarily offline.
+
+# Can Pure Off-Policy Agents Succeed?
+
+The authors first trained a standard Nature DQN and saved every interaction with the environment, which became the dataset $$\mathcal{B}_{DQN}$$.
```

Comment:
Does "Nature DQN" is common for referring to DeepMind's DQN work?

Reply:
Yes